go.mod

```diff
@@ -3,14 +3,13 @@ module github.com/ava-labs/teleporter
 go 1.21.12
 
 require (
-	github.com/ava-labs/avalanchego v1.11.1
+	github.com/ava-labs/avalanchego v1.11.9-0.20240703222923-9d71967ccbba
```

Comment:
`9d719...` refers to the latest commit hash (as of this writing) in [this PR](https://github.com/ava-labs/avalanchego/pull/3179). We should change this to refer to a released version, once that PR has been merged and released.

Reply:
This is probably what's blocking the e2e tests running

Reply:
nvm. I was lacking context on the issue being around conflicting dependencies on avalanchego.